test/unit/generator/content/part2/get-table/get-table.test.js

```diff
@@ -0,0 +1,60 @@
+const getTable = require('../../../../../../app/generator/content/part2/get-table')
+let mockStatement
+let reductions
+
+describe('get table', () => {
+  beforeEach(() => {
+    mockStatement = JSON.parse(JSON.stringify(require('../../../../../mock-statement-data')))
+    reductions = [{ id: 1, reason: 'Late claim submission' }, { id: 2, reason: 'Over declaration reduction' }]
```

Comment:
pull out into own mock object sim. to `mock-statement-data`

used in multiple files and we can see the benefit:

- in here and `get-table/map-funding-rows.test.js` they resemble real-life examples
- but in `get-table/map-reduction-ids.test.js` they're placeholder
- not to mention making the tests familiar, predictable and orientated to the business process/output